tests/e2e/performance/test_pod_reattachtime.py

```diff
@@ -62,85 +140,128 @@ def test_pvc_reattach_time_performance(self, pvc_factory, teardown_factory):
+
+            rsh_cmd = (
+                f"exec {pod_name} -- tar xvf {pod_path}/tmp/file.gz -C {pod_path}/tmp"
+            )
             _ocp.exec_oc_cmd(rsh_cmd)
-            rsh_cmd = f"exec {pod_name} -- sync"
+
+            for x in range(copies):
+                rsh_cmd = f"exec {pod_name} -- mkdir -p {pod_path}/folder{x}"
+                _ocp.exec_oc_cmd(rsh_cmd)
+                rsh_cmd = (
+                    f"exec {pod_name} -- cp -r {pod_path}/tmp {pod_path}/folder{x}"
+                )
+                _ocp.exec_oc_cmd(rsh_cmd)
+                rsh_cmd = f"exec {pod_name} -- sync"
+                _ocp.exec_oc_cmd(rsh_cmd)
+
+            logging.info("Getting the amount of data written to the PVC")
+            rsh_cmd = f"exec {pod_name} -- df -h {pod_path}"
+            data_written = _ocp.exec_oc_cmd(rsh_cmd).split()[-4]
+            logging.info(
+                f"The Amount of data that was written to the pod is {data_written}"
+            )
+            rsh_cmd = f"delete pod {pod_name}"
             _ocp.exec_oc_cmd(rsh_cmd)
 
-        log.info("Getting the amount of data written to the PVC")
-        rsh_cmd = f"exec {pod_name} -- df -h {pod_path}"
-        data_written = _ocp.exec_oc_cmd(rsh_cmd).split()[-4]
-        log.info(f"The Amount of data that was written to the pod is {data_written}")
-        rsh_cmd = f"delete pod {pod_name}"
-        _ocp.exec_oc_cmd(rsh_cmd)
-
-        logging.info(f"Creating Pod with pvc {pvc_obj.name} on node {node_two}")
-
-        pod_obj2 = helpers.create_pod(
-            interface_type=self.interface,
-            pvc_name=pvc_obj.name,
-            namespace=pvc_obj.namespace,
-            node_name=node_two,
-            pod_dict_path=constants.PERF_POD_YAML,
-        )
+            logging.info(f"Creating Pod with pvc {pvc_obj.name} on node {node_two}")
 
-        start_time = time.time()
+            pod_obj2 = helpers.create_pod(
+                interface_type=self.interface,
+                pvc_name=pvc_obj.name,
+                namespace=pvc_obj.namespace,
+                node_name=node_two,
+                pod_dict_path=constants.PERF_POD_YAML,
+            )
 
-        pod_name = pod_obj2.name
-        helpers.wait_for_resource_state(
-            resource=pod_obj2, state=constants.STATUS_RUNNING, timeout=120
-        )
-        end_time = time.time()
-        total_time = end_time - start_time
-        if total_time > 60:
-            raise ex.PerformanceException(
-                f"Pod creation time is {total_time} and greater than 60 seconds"
+            start_time = time.time()
+
+            pod_name = pod_obj2.name
+            helpers.wait_for_resource_state(
+                resource=pod_obj2, state=constants.STATUS_RUNNING, timeout=120
+            )
+            end_time = time.time()
+            total_time = end_time - start_time
+            if total_time > 60:
+                raise ex.PerformanceException(
```

Comment:
it is good to add here log.error() message

Reply:
Done, thank you.